.github/workflows/backend.yml

```diff
@@ -23,13 +23,48 @@ jobs:
   test:
     name: pytest
     runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:13.4
```

Comment:
Should this use 15 instead of 13? We're using 15.6 in production, and I'm surprised to still see 13.4 [here](https://github.com/mozilla/pontoon/blob/ee29b881b24d6382ec6fbbad84d3779cd4c5b105/docker/postgres/Dockerfile#L1) (but 15 [here](https://github.com/mozilla/pontoon/blob/ee29b881b24d6382ec6fbbad84d3779cd4c5b105/docker/Dockerfile#L28)).

Reply:
Good catch, let's use PG 15 everywhere.

Reply:
```suggestion
        image: postgres:15
```